### ISSUE TITLE
Is the aws_cognito_user_pool resource available?

### ISSUE THREAD

**Issue by hashibot** [terraform-providers/terraform-provider-aws#2900]
_This issue was originally opened by @dsalazar-carecloud as hashicorp/terraform#17062. It was migrated here as a result of the [provider split](https://www.hashicorp.com/blog/upcoming-provider-changes-in-terraform-0-10/). The original body of the issue is below._

<hr>

Hello I was just wondering if the **aws_cognito_user_pool** resource is available for use?  Excited to use it since I noticed it in your docs but unfortunately it's yielding the following for me.
```
Error: aws_cognito_user_pool.foo: Provider doesn't support resource: aws_cognito_user_pool
```
```
Terraform v0.11.1
+ provider.aws v1.2.0
+ provider.template v1.0.0
```
```hcl
resource "aws_cognito_user_pool" "foo" {
  name = "Foo"
}
```

**Comment by dsalazar-carecloud** [comment]
Apologies.  Just read a bit more of the documentation and noticed that modules can be upgraded with the terraform init --upgrade command.

**Comment by bflad** [comment]
@dsalazar-carecloud glad you were able to figure out upgrading your provider version. For other future travelers, documentation is available here on the subject including version pinning in Terraform configurations: https://www.terraform.io/docs/configuration/providers.html#provider-versions

As for the `aws_cognito_user_pool` resource documentation (available in AWS provider version 1.3.0+) that is available here: https://www.terraform.io/docs/providers/aws/r/cognito_user_pool.html

I'll close this issue out since it seems like your issue is resolved, but please don't hesitate to let us know if you need anything else. 😄
